app/assets/stylesheets/app/froggo_teams.scss

```diff
@@ -75,13 +75,36 @@
     font-family: $main-font;
   }
 
-  &__organization-title {
-    align-items: center;
+  &__organization-teams-section {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    padding: 15px;
+  }
+
+  &__teams-section {
+    flex: 1;
+    &:not(:first-child) {
+      margin-left: 40px;
+    }
+  }
+
+  &__belonged-teams-title {
+    color: $primary-color;;
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected newline after ";" in a multi-line declaration block   declaration-block-semicolon-newline-after